cli/targets/static.js

```diff
@@ -354,8 +354,48 @@ function toJsType(field) {
     return type;
 }
 
+function syntaxForType(type) {
+
+    var syntax = null;
+    var namespace = type;
+
+    while (syntax === null && namespace !== null) {
+        if (namespace.options != null && "syntax" in namespace.options)
+            syntax = namespace.options["syntax"];
+        else
+            namespace = namespace.parent;
+    }
+
+    return (syntax === "proto3") ? "proto3" : "proto2";
```

Comment:
With editions this will likely be more options, could we make this `return syntax ?? proto2`

Reply:
Done, changes pushed